packages/synapse-react-client/src/components/useImmutableTableQuery.ts

```diff
@@ -302,7 +404,13 @@ export default function useImmutableTableQuery(
   )
 
   const removeValueFromSelectedFacet = useCallback(
-    (facet: FacetColumnRequest, value: string) => {
+    (
+      facet: FacetColumnRequest | string,
```

Comment:
There's at most one facet per column, so allow passing a column name to make it easier.

The motivation for this change is that the facet filter controls are populated using the facet statistics response, not the `FacetColumnRequest` objects. Within the control components, we can make facet manipulation very simple by just performing operations against a column name--the complexity of the `FacetColumnRequest` objects can be abstracted.

Reply:
Excellent